pkg/nodeagent/runtime/fake_runtime.go

```diff
@@ -0,0 +1,179 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package runtime
+
+import (
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	criv1 "k8s.io/cri-api/pkg/apis/runtime/v1"
+)
+
+var _ RuntimeService = &FakeRuntimeService{}
```

Comment:
is this Fake Runtime Service used somewhere? is it purposed for test use only?

Reply:
for test only later use